variables.tf

```diff
@@ -4,6 +4,12 @@ variable "ami_id" {
   default     = ""
 }
 
+variable "ami_architecture" {
+  type        = string
+  description = "Architecture of the Spacelift AMI. Currently, only x86_64 or arm64 are supported."
```

Comment:
Can you add a `validation` block here to make sure that only these 2 values can be passed in? 🙏